RxJavaTemplate[DEPRECATED]/build.gradle.kts

```diff
@@ -21,7 +21,7 @@ plugins {
 allprojects {
     repositories {
         google()
-        maven(url = "https://jitpack.io")
+        maven { url = uri("https://www.jitpack.io") }
```

Comment:
@lydiasama Since this package is deprecated, I'm not sure we should do the update or not 😄

Reply:
@doannimble I'll keep updating until @ryan-conway said Stop it! 😂

Reply:
@luongvo I'm not sure if I understand your expectation correctly, but do you have another intention outside of fixing this error by fetching Chucker from MavenCentral?

Reply:
@lydiasama No, just my concern about fetching the dependency from the recommended source.

Reply:
@luongvo My concern is if we get rid of maven Jitpack, will it affect to another lib we use?

Reply:
@luongvo @ryan-conway The PR is here. https://github.com/nimblehq/android-templates/pull/487